### PR TITLE
[TASK] Revise TSFE section

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main, 11.5

### CODE REVIEW

File: Documentation/AppendixA/Index.rst [hunk 1]
```diff
@@ -127,17 +127,17 @@ Internal variables in the main frontend object, TSFE
 
 There are some variables in the global object, TSFE (TypoScript
 Frontend), you might need to know about. These ARE ALL READ-ONLY!
-(Read: Don't change them!) See the class :php:`TypoScriptFrontendController`
-for the full descriptions.
+(Read: Don't change them!) See the class
```
Comment (linawolf) [inline Documentation/AppendixA/Index.rst:130]:
```suggestion
(Read: Do not change them!) See the class
```

File: Documentation/AppendixA/Index.rst [hunk 2]
```diff
@@ -127,17 +127,17 @@ Internal variables in the main frontend object, TSFE
...
+:php:`\TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController` for the
+full descriptions.
 
-.. important::
+You can retrieve the :php:`TypoScriptFrontendController` via the
+:ref:`request <typo3-request>` attribute
```
Comment (linawolf) [inline Documentation/AppendixA/Index.rst:135]: These are found in TYPO3 Explained? The prefix would be needed for the link then.

File: Documentation/AppendixA/Index.rst [hunk 3]
```diff
@@ -263,7 +266,10 @@ Global variables
          object
 
    Description
-         Main frontend object.
+         Main frontend object. Whenever possible, use the
+         :ref:`request <typo3-request>` attribute
```
Comment (linawolf) [inline Documentation/AppendixA/Index.rst:270]: link to TYPO3 Explained? Link Prefix is missing

File: Documentation/AppendixA/Index.rst [hunk 4]
```diff
@@ -127,17 +127,17 @@ Internal variables in the main frontend object, TSFE
 
 There are some variables in the global object, TSFE (TypoScript
 Frontend), you might need to know about. These ARE ALL READ-ONLY!
-(Read: Don't change them!) See the class :php:`TypoScriptFrontendController`
-for the full descriptions.
+(Read: Don't change them!) See the class
+:php:`\TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController` for the
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/AppendixA/Index.rst:131]: No, then we have to link to main, but the docs are also merged into 11.5 (or vice versa). A developer/integrator should be able to look up a class.

File: Documentation/AppendixA/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/AppendixA/Index.rst:135]: You are right. Seems like sphinx found them if they are unique in the intersphinx universe (so my preview was okay).

**Pull request merged**